juniper-vpn.py

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env python3
+
+import warnings
+
+if __name__ == '__main__':
+    warnings.warn('Please use the shim created by pip when installing the junipervpn package. If you use a virtualenv, it will be in PATH after activating it')
+
+    # Import after the warning, so the warning is the first thing seen rather
+    # than some ImportError due to some missing packages
+    from junipervpn import vpn
```

Comment:
I think `tncc.py` is meant to be usable standalone too - that's probably why currently both scripts are in the top level, rather than having `tncc.py` in its own module.

Would you mind adding the shim in the same commit as the directory rearranging please?

Reply:
I'll add a shim and an entry point as well, since it's probably a legitimate use as well